packages/lib/index.ts

```diff
@@ -1 +1,5 @@
-export {};
+import { Role, User } from '@documenso/prisma/client';
```

Comment:
I'd consider moving this to something like `universal/guards/is-admin.ts` I've been avoiding barrel files since they can have some gnarly outcomes during bundling 😸